app/jobs/send_spouse_auth_docs_to_zendesk_job.rb

```diff
@@ -0,0 +1,14 @@
+class SendSpouseAuthDocsToZendeskJob < ApplicationJob
+  queue_as :default
+
+  def perform(intake_id)
+    intake = Intake.find(intake_id)
+
+    if intake.completed_intake_sent_to_zendesk
```

Comment:
we don't have an equivalent for `completed_intake_sent_to_zendesk` for sending the spouse documents. i don't know that it's necessary but wanted to call that out in case someone had an opinion

Reply:
seems unlikely